.cloud-build/execute_changed_notebooks_cli.py

```diff
@@ -129,26 +139,65 @@ def str2bool(v):
     base_branch=args.base_branch,
 )
 
+
+def _load_results() -> list:
```

Comment:
After addressing the `dict` comment, this should be`def _load_results() -> List[Dict[str, Any]]`.

Reply:
done